src/confluent_kafka/schema_registry/json_schema.py

```diff
@@ -47,16 +45,15 @@ def __exit__(self, *args):
 
 class JSONSerializer(Serializer):
     """
-    JsonSerializer serializes objects in the Confluent Schema Registry binary
-    format for JSON.
+    Serializer that outputs JSON formatted data with Confluent Schema Registry framing.
```

Comment:
nit: formatted -> encoded

Reply:
that is precisely the word i was looking for but didn't come to mind. thanks.